### PR TITLE
added SLEEP(x) scalar function for testing KILL statements

### PR DESCRIPTION
**Pull request by chaudum**

### CODE REVIEW

**Comment by dobe** [comment]
why don't you use the crate plugin mechanism?

**Comment by mfussenegger** [comment]
testKillSelect failed on travis

File: sql/src/test/java/io/crate/testing/SleepScalarFunction.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+    @SafeVarargs
+    @Override
+    public final Boolean evaluate(Input<Long>... args) {
+        long millis = (args.length > 0) ? args[0].value() : 1000L;
+        try {
+            Thread.sleep(millis);
+            return true;
+        } catch (InterruptedException e) {
+            e.printStackTrace();
```
Comment (mfussenegger) [inline sql/src/test/java/io/crate/testing/SleepScalarFunction.java]: remove printStackTrace

File: sql/src/test/java/io/crate/integrationtests/KillIntegrationTest.java [hunk 2]
```diff
@@ -123,4 +123,11 @@ public void testKillGroupBy() throws Exception {
         runJobContextReapers();
     }
 
+    @Test
+    public void testKillSelect() throws Exception {
+        assertGotCancelled("SELECT sleep(500) FROM sys.nodes", null);
+        setup.setUpEmployees();
+        assertGotCancelled("SELECT *, sleep(500) FROM employees", null);
+    }
```
Comment (dobe) [inline sql/src/test/java/io/crate/integrationtests/KillIntegrationTest.java]: would be nice to have 2 tests - something like "testKillSelectSysTable"

File: sql/src/test/resources/META-INF/services/io.crate.Plugin [hunk 3]
```diff
@@ -0,0 +1 @@
+io.crate.testing.plugin.SleepPlugin
```
Comment (dobe) [inline sql/src/test/resources/META-INF/services/io.crate.Plugin]: would be better to call it CrateTestingPlugin crate-testing-plugin

**Comment by dobe** [comment]
ack

**Pull request merged**